authentication/authentication-styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+}
+
+.card {
+    max-width: 50vw;
+    min-width: fit-content;
+    height: 50vh;
+    background-color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    border-radius: 10px;
+    top: 50%;
+    box-shadow: 5px 5px 10px grey;
+
+}
+
+p {
+    width: 80%;
+    text-align: center;
+    margin: auto;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-size: small;
+}
+
+.divider {
+    height: 1px;
+    background-color: grey;
+    width: 80%;
+    margin: auto;
+    margin-top: 50px;
+}
+
+git a {
+    color: lightblue;
+}
+
+input {
+    display: block;
+    margin: auto;
+    width: 80%;
+    height: 30px;
+    border-radius: 5px;
+    box-sizing: content-box;
+    margin-bottom: 10px;
+}
+
+.create-account-btn {
+    background-color: rgb(255, 166, 0);
+    height: 40px;
```

Comment:
Instead of using height, we should use padding and leave the height be. By this way, the button text will not be overflow if we change the text size

```diff
+ padding-top: 16px;
+ padding-bottom: 16px;
- height: 40px;
```